docs/tfengine/schemas/devops.md

```diff
@@ -45,7 +45,19 @@ Type: string
 
 ### admins_group.owners
 
-Owners of the group.
+Owners of the admins group.
+
+Type: array(string)
+
+### admins_group.managers
+
+Managers of the admins group.
+
+Type: array(string)
+
+### admins_group.members
+
+Members of the admins group.
```

Comment:
For these ones, you can add them to the schema and regen, then they will be added to the md files. Nice catch!

Reply:
Hmmm. I think for these two specifically, they are commented out in the schema. And that's intentional, so you can leave them as is.

Reply:
Found them. I wasn't aware of the `regen` script when I made this PR. Most of the changes were done manually. I'm resolving this since now I saw them commented and the reason they're not available.

Any further questions I'll open another thread